src/streamlink/plugins/twitch.py

```diff
@@ -229,8 +229,17 @@ class TwitchHLSStreamReader(HLSStreamReader):
 class TwitchHLSStream(HLSStream):
     def __init__(self, *args, **kwargs):
         HLSStream.__init__(self, *args, **kwargs)
-        self.disable_ads = self.session.get_plugin_option("twitch", "disable-ads")
-        self.low_latency = self.session.get_plugin_option("twitch", "low-latency")
+
+        disable_ads = self.session.get_plugin_option("twitch", "disable-ads")
+        low_latency = self.session.get_plugin_option("twitch", "low-latency")
+
+        if low_latency and disable_ads:
+            log.info("Low latency streaming with ad filtering is currently not supported")
```

Comment:
it is not really clear in this message, which option is used and which is disabled.

Reply:
True, but both options print separate info log lines when set, so it's not important if this line doesn't tell which one gets disabled.